sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulsePingSender.java

```diff
@@ -162,7 +152,7 @@ private String buildPingEntity(long timeInMillis) throws JsonProcessingException
             pingEnvelope.setVersion(sdkVersion);
         }
         pingEnvelope.setTimeStamp("/Date(" + timeInMillis + ")/");
-        return mapper.writeValueAsString(pingEnvelope);
+        return pingEnvelope.toJsonString().replace("/", "\\/");
```

Comment:
can you add this comment here 
`// By default jackson doesn't escape forward slashes (`/`), but the quick pulse backend requires
 // them to be escaped.`

Reply:
comment was from deleted `CustomCharacterEscapes`.